README.md

```diff
@@ -1,2 +1,248 @@
-# web-devops-roadmap
-DevOps Roadmap for web developers
+# Web DevOps Roadmap
+
+This repository contains initial DevOps roadmap for all the web developers. Anyone can start this training with their existing learning projects. The approximate time for complete these practicals is 10 days.
+
+# Guidelines
+- Before starting any practice, it's essential to conduct research and learn the necessary concepts.
+
+- Create a GitHub/GitLab repository for each Practical if one doesn't already exist.
+
+- Learn Basic DevOps Principles:
+  - Learn how to use Version Control effectively for your codebase as Git is Fundamental.
+  - Know how to write unit tests.
+  - Start by understanding the core DevOps concepts like continuous integration, continuous deployment, and automation.
+  - Learn some basic scripts to get a basic understanding of scripting language.
+  - Be aware of basic security practices like exposing env vars, Inbound/Outbound security rules, etc…
+
+- Note for AWS servies:
+  - Starting by manually deploying application through the AWS console can provide a solid foundational understanding of the service. 
```

Comment:
Need to mention other clouds as well, making AWS default for our organization.
- Start deploying an application manually through the AWS /Gcloud/Azure console. It will provide a solid foundational understanding of the service.